libraries/core/src/main/kotlin/com/bumble/appyx/core/navigation/NavModelAdapter.kt

```diff
@@ -9,6 +9,7 @@ interface NavModelAdapter<NavTarget, State> {
     val screenState: StateFlow<ScreenState<NavTarget, out State>>
 
     data class ScreenState<NavTarget, State>(
+        val targetStateVisible: NavElements<NavTarget, out State> = emptyList(),
```

Comment:
Can we add more info about what this `targetStateVisible` represents? It's a bit out of context in the `ScreenState` class. 

`onScreen` and `offScreen` makes sense but not `targetStateVisible`. How's that state supposed to mutate/evolve over time?

Reply:
yeah, it conceptually belongs to the `onScreen` element. We need to know which elements from the `onScreen` list are transitioning to `visibleTargetState` but it will break the API if I make a wrapper class

Reply:
Gotcha! Can we be more explicit then? `onScreenVisibleTargetState` perhaps?